test/sql/chunk_merge.sql

```diff
@@ -0,0 +1,54 @@
+-- This file and its contents are licensed under the Apache License 2.0.
```

Comment:
This file seems misplaced since compression is TLS-feature the test should  be in tsl folder and not mixed with apache-code